bakery/src/pipelines/gdoc.js

```diff
@@ -0,0 +1,104 @@
+const pipeline = (env) => {
+  const taskCheckFeed = require('../tasks/check-feed')
+  const taskDequeueBook = require('../tasks/dequeue-book')
+  const taskFetchBook = require('../tasks/fetch-book')
+  const taskAssembleBook = require('../tasks/assemble-book')
+  const taskAssembleBookMeta = require('../tasks/assemble-book-metadata')
+  const taskBakeBook = require('../tasks/bake-book')
+  const taskBakeBookMeta = require('../tasks/bake-book-metadata')
+  const taskChecksumBook = require('../tasks/checksum-book')
+  const taskDisassembleBook = require('../tasks/disassemble-book')
+  const taskJsonifyBook = require('../tasks/jsonify-book')
+  const taskValidateXhtml = require('../tasks/validate-xhtml')
+
+  const awsAccessKeyId = env.ENV_NAME === 'local' ? env.S3_ACCESS_KEY_ID : '((aws-sandbox-secret-key-id))'
+  const awsSecretAccessKey = env.ENV_NAME === 'local' ? env.S3_SECRET_ACCESS_KEY : '((aws-sandbox-secret-access-key))'
+  const codeVersionFromTag = env.IMAGE_TAG || 'version-unknown'
+  const queueFilename = `${codeVersionFromTag}.${env.QUEUE_FILENAME}`
```

Comment:
General question related to the combination of `S3_QUEUE_STATE_BUCKET` and `QUEUE_FILENAME` in this pipeline. Do we want the queue files for gdoc to live in a separate bucket from web-hosting builds, or the same bucket but different name structure for the file, or some combination of both, or ...?

Reply:
Seems like there's no specific decision, so I left it as is, since we determined the functionality will at least be similar. I am aiming to kick the can down the road on this one.